### PR TITLE
add pytest-xprocess 0.18.1

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Notes:
- test dependency of https://github.com/conda-forge/werkzeug-feedstock

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge/staged-recipes @conda-forge/help-python ready for review!

File: recipes/pytest-xprocess/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  requires:
+    - pip
+    - coverage
+  imports:
+    - pytest_xprocess
+    - xprocess
+  commands:
+    - pip check
+    - coverage run -m pytest -v
+    - coverage report --fail-under={{ cov_fail_under }}
```
Comment (ocefpaf) [inline recipes/pytest-xprocess/meta.yaml:39]: I'm not sure running coverage tests in the feedstock are worth our cycles. This is something for devs upstream, we are not really changing the code (much) here.

Comment (bollwyvl) [reply recipes/pytest-xprocess/meta.yaml:39]: happy to have the broader discussion, but my motivation has been, especially on more complicated, multi-output packages, that the test suite is good at finding underlying required dependency issues, while coverage helps find drastic optional dependency changes that might trigger a new output.

**Comment by bollwyvl** [comment]
Thanks!

File: recipes/pytest-xprocess/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipes/pytest-xprocess/meta.yaml:39]: CI resources are OK now. My old mind is still plagued by our CircleCI/Travis-CI times. It is probably OK to run that in the feedstock.

Comment (bollwyvl) [reply recipes/pytest-xprocess/meta.yaml:39]: yeah, maybe there's a reasonable threshold under which, it's fine if do more validation. most noarch python would fall there... but being picky has definitely resulted in some upstream fixes.

**Pull request merged**